src/plugin/arc/layer/arclayerdescriptor.js

```diff
@@ -554,3 +561,38 @@ plugin.arc.layer.ArcLayerDescriptor.prototype.getNodeUI = function() {
 plugin.arc.layer.ArcLayerDescriptor.prototype.getFilterableTypes = function() {
   return [this.getId() + os.ui.data.BaseProvider.ID_DELIMITER + 'features'];
 };
+
+/**
+ * @inheritDoc
+ */
+plugin.arc.layer.ArcLayerDescriptor.prototype.getUrlKey = function() {
```

Comment:
And not add this to the implementation. See https://github.com/ngageoint/opensphere/issues/545

Reply:
Ditto to interface comment.